src/Refactoring2-Transformations/RBInsertNewClassTransformation.class.st

```diff
@@ -1,35 +1,16 @@
-"
-Adds a class in the image, optionally inside an hierarchy (with superclass or subclasses).
-
-Usage:
-| transformation |
-transformation := (RBInsertClassTransformation
-	addClass: #InsertedClass
-	superclass: #RBTransformationTest
-	subclasses: (Array with: RBInsertClassTransformationTest)
-	category: #'Refactoring2-Transformations-Tests')
-	transform.
-(ChangesBrowser changes: transformation model changes changes) open
-
-Preconditions:
-- superclass must be an existing class
-- each subclass also must be an existing class
-- name of the class to be added must be non-existent in the system
-- name of the category must be a valid one
-"
```

Comment:
There is a bug currently in Pharo 12 that makes some comments vanish. And it happened here.

(A fix for this bug was just integrated)

Reply:
Nice catch @jecisc! I've just returned the removed comment. Glad that was fixed, it was a nasty bug.